bftengine/src/bcstatetransfer/Messages.hpp

```diff
@@ -156,6 +156,7 @@ struct ItemDataMsg : public BCStateTranBaseMsg {
   uint16_t chunkNumber;
 
   uint32_t dataSize;
+  uint lastInBatch;
```

Comment:
bool?

Reply:
yes, it's a bool, but bool is implementation-specific. can be 1 byte or 4 bytes. It's not a good habit in my idea to put a bool on messages going between machines.

Reply:
can you change it to a more standart uint8_t?

Reply:
Sure, That is a typo, good that you saw it.
I meant it to be uint8_t.
Thanks.